server_action_navigate/models/ir_actions_server_navigate_line.py

```diff
@@ -22,7 +22,10 @@ class IrActionsServerNavigateLine(models.Model):
     )
 
     field_id = fields.Many2one(
-        comodel_name="ir.model.fields", string="Field", required=True
+        comodel_name="ir.model.fields",
+        string="Field",
+        required=True,
+        ondelete="cascade",
```

Comment:
I think this ondelete will create unconsistencies state and so doesn't see the interest...

Reply:
> I think this ondelete will create unconsistencies state and so doesn't see the interest...

Odoo is not allowing this field without ondelete cascade, and shouldn't be. Because when this field gets deleted, the navigate line also should get deleted. Navigate line has no meaning without that field, just like the sale order line has no meaning without the sale order.

Reply:
Agree with you. It wasn't my point.
consider a server action navigate of (sale.order) with 4 lines : 
- line 1, field 1 (line_ids), model 1 (sale.order.line)
- line 2, field 2 (product_id), model 2 (product.product)
- line 3, field 3 (product_tmpl_id) model 3 (product.template)
- line 4, field 4 (categ_id), model 4 (product.category)

If something remove the field 2 & 3, it will remove the line 2 & 3 . so the final configuration will be :
- line 1, field 1 (line_ids), model 1 (sale.order.line)
- line 4, field 4 (categ_id), model 4 (product.category)

that will be inconsistent, because ``categ_id`` is not a field of ``sale.order.line``.

That being said, I suspect that it is a theoritical case, because if field 3 is uninstalled, field 4 should also uninstalled, by dependency.

So, sorry for the noice on that topic. :+1:

Reply:
ok, I got the problem, and yes it is a theoritical case and may happen rarely. So what do you suggest? Should I add the fix for it?